tinker/libs/bunq_lib.py

```diff
@@ -52,18 +54,29 @@ def setup_context(self):
         else:
             raise BunqException(self._ERROR_COULD_NOT_DETIRMINE_CONF)
 
-        api_context = ApiContext.restore(self.determine_bunq_conf_filename())
-        api_context.ensure_session_active()
-        api_context.save(self.determine_bunq_conf_filename())
+        try:
+            api_context = ApiContext.restore(self.determine_bunq_conf_filename())
+            api_context.ensure_session_active()
+            api_context.save(self.determine_bunq_conf_filename())
 
-        BunqContext.load_api_context(api_context)
+            BunqContext.load_api_context(api_context)
+        except ForbiddenException as forbidden_exception:
+            self.handle_forbidden_exception(forbidden_exception)
 
     def determine_bunq_conf_filename(self):
         if self.env == ApiEnvironmentType.PRODUCTION:
             return self._BUNQ_CONF_PRODUCTION
         else:
             return self._BUNQ_CONF_SANDBOX
 
+    def handle_forbidden_exception(self, forbidden_exception):
+        if self.env == ApiEnvironmentType.SANDBOX \
+                and self._ERROR_INSUFFICIENT_AUTHENTICATION in forbidden_exception.message:
```

Comment:
regex 😁  consistency across all the repos 😉. To check if a string is within another string === regex. Or check if both strings are the same. Consistency across all the repos makes it easier to maintain.